ops/helper-scripts/accessibility-test.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -e
+set -euo pipefail
```

Comment:
I don't generally suggest it, but should we add a comment here, or since this is used in several files maybe add a description of what this does and when/why to use it to our documentation?

Reply:
These were added to properly fail the running job step when an error occurred. Before, was seeing failed azure cli commands but the subsequent steps continue to run.

Reply:
Added a comment for that line